area/area.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	fmt.Println("Enter three triangle sides:")
+	var a, b, c float64
+	fmt.Scanln(&a, &b, &c)
+	if a+b > c && b+c > a && c+a > b {
+		S := (a + b + c) / 2
```

Comment:
nit: Please consider using lowercase letters for variable names e.g. `s` and `area`.